grammars.yml

```diff
@@ -63,7 +63,7 @@ vendor/grammars/NSIS:
 - source.nsis
 vendor/grammars/NimLime:
 - source.nim
-- source.nim_filter
+- source.nimscf
```

Comment:
Don't manually change this file. It's automatically updated based on the files in each grammar.

Reply:
done.